From e683c0f6f1b15f9135ed1d8418dabd753d09783d Mon Sep 17 00:00:00 2001
From: Mike FABIAN <mfabian@redhat.com>
Date: Wed, 13 May 2015 08:56:45 +0200
Subject: [PATCH 4/8] Translation fix for Tagalog <-> Filipino
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Resolves: rhbz#1220775 - “dnf langavailable” lists “fil” as available but “dnf langavailable fil” then says it is not available
See: https://bugzilla.redhat.com/show_bug.cgi?id=1220775

Current CLDR has

.../cldr-svn/trunk/common/main
$ grep 'language.*fil'  so.xml
			<language type="fil" draft="unconfirmed">Tagalog</language>

Which is probably not right, Tagalog seems appropriate for "tl",
it is not nice if "fil" has exactly the same translation.
---
 data/languages.xml | 4 ++--
 test_cases.py      | 6 ++++++
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/data/languages.xml b/data/languages.xml
index b9c56a1..d4f80fb 100644
--- a/data/languages.xml
+++ b/data/languages.xml
@@ -6280,7 +6280,6 @@
       <name><languageId>si</languageId><trName>පිලිපීන</trName></name>
       <name><languageId>sk</languageId><trName>filipínčina</trName></name>
       <name><languageId>sl</languageId><trName>filipinščina</trName></name>
-      <name><languageId>so</languageId><trName>Tagalog</trName></name>
       <name><languageId>sq</languageId><trName>filipinase</trName></name>
       <name><languageId>sr</languageId><trName>филипински</trName></name>
       <name><languageId>sr_Latn</languageId><trName>filipinski</trName></name>
@@ -6292,7 +6291,6 @@
       <name><languageId>tg</languageId><trName>Филиппинӣ</trName></name>
       <name><languageId>th</languageId><trName>ฟิลิปปินส์</trName></name>
       <name><languageId>ti</languageId><trName>ታጋሎገኛ</trName></name>
-      <name><languageId>tn</languageId><trName>Tagalog</trName></name>
       <name><languageId>tr</languageId><trName>Filipince</trName></name>
       <name><languageId>ug</languageId><trName>فىلىپپىنچە</trName></name>
       <name><languageId>uk</languageId><trName>філіппінська</trName></name>
@@ -21727,6 +21725,7 @@
       <name><languageId>ru</languageId><trName>тагалог</trName></name>
       <name><languageId>sk</languageId><trName>tagalčina</trName></name>
       <name><languageId>sl</languageId><trName>tagalogščina</trName></name>
+      <name><languageId>so</languageId><trName>Tagalog</trName></name>
       <name><languageId>sr</languageId><trName>Тагалски</trName></name>
       <name><languageId>sr_Latn</languageId><trName>Tagalski</trName></name>
       <name><languageId>sv</languageId><trName>tagalog</trName></name>
@@ -21735,6 +21734,7 @@
       <name><languageId>th</languageId><trName>ตากาล็อก</trName></name>
       <name><languageId>tig</languageId><trName>ታጋሎገኛ</trName></name>
       <name><languageId>tl</languageId><trName>Tagalog</trName></name>
+      <name><languageId>tn</languageId><trName>Tagalog</trName></name>
       <name><languageId>tr</languageId><trName>Takalotça</trName></name>
       <name><languageId>ug</languageId><trName>تاگالوگچە</trName></name>
       <name><languageId>uk</languageId><trName>тагальська</trName></name>
diff --git a/test_cases.py b/test_cases.py
index 667c293..744dc61 100644
--- a/test_cases.py
+++ b/test_cases.py
@@ -1222,6 +1222,12 @@ def dummy():
     >>> languageId("Low German (Germany)")  # doctest: +NORMALIZE_WHITESPACE
         'nds_DE'
 
+    >>> languageId("Tagalog")  # doctest: +NORMALIZE_WHITESPACE
+        'tl'
+
+    >>> languageId("Filipino")  # doctest: +NORMALIZE_WHITESPACE
+        'fil'
+
     >>> print(langtable.timezone_name(timezoneId='US/Mountain', languageIdQuery='ja'))  # doctest: +NORMALIZE_WHITESPACE
        アメリカ合衆国/山地時間
 
-- 
2.4.3

